docs/using/syntax.md

```diff
@@ -451,11 +451,16 @@ Target headers are defined with this syntax:
 (header_target)=
 ```
 
-They can then be referred to with the "ref" inline role:
+They can then be referred to with the [ref](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-ref) inline role:
```

Comment:
Could we expand these links to encompass the "inline role" words as well? I find that single-word URLs are often hard for readers to notice

Reply:
Fixed